sdk/appconfiguration/app-configuration/rollup.base.config.js

```diff
@@ -40,8 +40,9 @@ export function nodeConfig(test = false) {
     // different output file
     baseConfig.output.file = "test-dist/index.node.js";
 
-    // mark assert as external
+    // mark assert packages we use as external
     baseConfig.external.push("assert");
+    baseConfig.external.push("chai");
```

Comment:
For the node.js config, you can probably just mark everything in `dependencies` and `devDependencies` as external since we never bundle our dependencies in the node version.

Reply:
Done.